app/database.py

```
@@ -0,0 +1,23 @@
+    def __init__(self,db_url,min_connections=2, max_connections=5, **kwargs) -> None:
+        self.db_url=db_url
+        self.max_connections=max_connections
+        self.min_connections=min_connections
+        self.db_engine=None
+        self.connection_params=kwargs
+        self.session=None
+        self.initialize_engine()
+        
+    def initialize_engine(self):
+        try:
+            if self.db_engine==None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)